app/main.py

```diff
@@ -1,2 +1,98 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+INITIAL_CAPACITY = 8
+RESIZE_THRESHOLD = 0.62
+CAPACITY_MULTIPLIER = 2
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            capacity:
```

Comment:
You shouldn't pass these params on init - they are part of dict internal structure